### DISCUSSION TITLE
Lerna link still required

### DISCUSSION THREAD

**Discussion by The-Code-Monkey** [lerna/lerna#3737] [Q&A]
Hi there,

I'm pretty new to using yarn workspaces and i can't seem to use it in the same way that I would have used lerna in the past.

So previously I would build my compiler cli tool and then use lerna link to symlink it to the rest of the packages that needed it.

But since V7 lerna no longer includes link by default, I cannot do this, I've tried to work out the workspace way of doing it, but it doesn't seem to link the package so, all the other packages fail due to the cli command missing from the .bin folder.

I have resorted to installing the @lerna/link package instead. 

Does anyone know how to get around this.

Here is the affected repo: https://github.com/The-Code-Monkey/TechStack

**Comment by fahslaj** [comment]
Hi @The-Code-Monkey , thank you for providing the affected repo. I was able to reproduce the issue. This seems to be a quirk with `yarn install` as described in [this yarn issue](https://github.com/yarnpkg/yarn/issues/4964#issuecomment-358709008). I was able to fix it in my local TechStack repo by deleting `node_modules` and running `yarn install` again. Then, `yarn build` completes successfully after removing the `@lerna/link` package. I would suggest trying this and seeing if that solves the problem for you.
